legal-api/tests/unit/services/filings/validations/change_of_director/test_validation_effective_date.py

```diff
@@ -35,11 +26,14 @@
 
 def common_setup(identifier: str, now: datetime):
     """Set up the data for COD tests."""
+    founding_date = now - datedelta.YEAR
     filing_date = now - datedelta.DAY
 
+    # create business founded "a year ago"
     business = Business(identifier=identifier,
-                        founding_date=now - datedelta.YEAR)
+                        founding_date=founding_date)
```

Comment:
is this the only place founding_date gets used, why the formatting change?

Reply:
I was being explicit (incl comment) about exactly what this variable means. (Eg, "now" isn't really now; it's a relative time within this function only. A bit confusing? Yes.)

I know it's valid syntax to put an expression as a function parameter, but I find that sometimes hides the meaning of the parameter. IMO this code is more clear.

I can revert if you feel strongly about this.